docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,82 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+List all the `tables` from the database with `database-name` and `is temporary table`.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} database_name] [LIKE 'regex_pattern']
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>{FROM|IN} database_name</em></code></dt>
+  <dd>
+     Specifies the `database` name from which tables are listed.
+  </dd>
+  <dt><code><em>LIKE 'regex_pattern'</em></code></dt>
+  <dd>
+     Specifies the regex pattern that is used to filter out unwanted tables.
+       <br> - Only `*` and `|` are allowed as wildcard pattern.
+       <br> - Excluding `*` and `|` the remaining pattern follows the regex semantics.
+       <br> - The leading and trailing blanks are trimmed in the input pattern before processing.  
+  </dd>
+</dl>
+
+### Example
+{% highlight sql %}
+-- List all tables in default database
+SHOW TABLES;
+  +-----------+------------+--------------+--+
+  | database  | tableName  | isTemporary  |
+  +-----------+------------+--------------+--+
+  | default   | sam        | false        |
+  | default   | sam1       | false        |
+  | default   | suj        | false        |
+  +-----------+------------+--------------+--+
+
+-- List all tables from userdb database 
+SHOW TABLES FROM userdb;
+  +-----------+------------+--------------+--+
+  | database  | tableName  | isTemporary  |
+  +-----------+------------+--------------+--+
+  | userdb    | user1      | false        |
+  | userdb    | user2      | false        |
+  +-----------+------------+--------------+--+
+
+-- List all tables in userdb database
+SHOW TABLES IN userdb;
+  +-----------+------------+--------------+--+
+  | database  | tableName  | isTemporary  |
+  +-----------+------------+--------------+--+
+  | userdb    | user1      | false        |
+  | userdb    | user2      | false        |
+  +-----------+------------+--------------+--+
+
+-- List all tables from default database matching the pattern `sam*`
+SHOW TABLES FROM default LIKE 'sam*';
+  +-----------+------------+--------------+--+
+  | database  | tableName  | isTemporary  |
+  +-----------+------------+--------------+--+
+  | default   | sam        | false        |
+  | default   | sam1       | false        |
+  +-----------+------------+--------------+--+
+-- List all tables matching the pattern `sam*|suj`
```

Comment:
Nit: can we have a line between previous test and this one.

Reply:
@dilipbiswal 
Done, updated the snap's also.